modules/natgw/.header.md

```diff
@@ -26,7 +26,7 @@ and Subnets):
 
 ```hcl
 module "natgw" {
-  source = "PaloAltoNetworks/vmseries-modules/azurerm//modules/natgw"
+  source = "PaloAltoNetworks/swfw-modules/azurerm//modules/natgw"
```

Comment:
We have the following format in some of the modules, and this one as well. Shall we unify all in one format?
```
  source = "../../modules/bootstrap"
```

Reply:
Hmm, I think we should use this format for any code snippets in the header.md files:
`source = "PaloAltoNetworks/swfw-modules/azurerm//modules/<module_name>"`

Reply:
I updated all of `source` for modules.